build.sh

```diff
@@ -589,34 +614,30 @@ function build_ios_target {
     ${BUILD_COMMAND}
 
     if [[ "${INSTALL_COMMAND}" ]]; then
-        echo "Installing ${lc_target} in out/${lc_target}/filament..."
+        echo "Installing ${lc_target} in out/${install_dir}/filament..."
         ${BUILD_COMMAND} ${INSTALL_COMMAND}
+        IOS_INSTALLED_LIBS_DIR="out/${install_dir}/filament/lib"
     fi
 
     cd ../..
 }
 
 function archive_ios {
-    local lc_target=$(echo "$1" | tr '[:upper:]' '[:lower:]')
+    local archive_suffix=$1
+    local archive_filename="filament-${archive_suffix}.tgz"
 
-    if [[ -d "out/ios-${lc_target}/filament" ]]; then
+    if [[ -d "${IOS_INSTALLED_LIBS_DIR}" ]]; then
         if [[ "${ISSUE_ARCHIVES}" == "true" ]]; then
-            echo "Generating out/filament-${lc_target}-ios.tgz..."
-            cd "out/ios-${lc_target}"
-            tar -czvf "../filament-${lc_target}-ios.tgz" filament
+            echo "Generating out/${archive_filename}..."
+            cd "${IOS_INSTALLED_LIBS_DIR}../../"
```

Comment:
So the output of `archive_ios` now depends on the parameters `build_ios_target` was last invoked with? This looks like a fragile dependency,  can't we jump to the folder directly like this? 
```sh
cd "out/${archive_suffix}-${lc_target}"
```
(Though after this, the name `archive_suffix` might need some rethinking, and the first parameter "Debug" or "Release" has to be brought back).

Reply:
74fd2bd3fffbecb601faa377ae5c761f5d0b8530 (I realized that I accidentally removed `debug`/`release` from the archive filename, so thanks for pointing ot)